src/test/java/uk/gov/hmcts/sptribs/common/notification/ContactPartiesNotificationTest.java

```diff
@@ -36,13 +40,15 @@ class ContactPartiesNotificationTest {
     private ContactPartiesNotification contactPartiesNotification;
 
     @Test
-    void shouldNotifySubjectOfCaseIssuedWithEmail() {
+    void shouldNotifySubjectOfContactPartiesWithEmailWithAttachments() {
```

Comment:
Only the methods sendToTribunal and sendToRespondent have a check for the ContactPartiesDocuments being empty or not. In the other methods if no attachment is provided the documentList just stays null. Since there is no check in the implementation not sure if a test is necessary for that. (test would be a duplicate almost).